src/main/webapp/dataverse.xhtml

```diff
@@ -1,527 +1,644 @@
-                                <div class="col-sm-9 form-col-container">
-                                    <p:inputText id="name" tabindex="1" styleClass="form-control" value="#{DataversePage.dataverse.name}"/>
-                                </div>
+      xmlns:jsf="http://xmlns.jcp.org/jsf"
+      xmlns:f="http://xmlns.jcp.org/jsf/core">
+<h:head>
+</h:head>
+
+<h:body>
+    <ui:composition template="/dataverse_template.xhtml">
+        <ui:param name="pageTitle"
+                  value="#{empty DataversePage.dataverse.name ? bundle.new : DataversePage.dataverse.name}"/>
+        <ui:param name="dataverse" value="#{DataversePage.dataverse}"/>
+        <ui:define name="body">
+            <f:metadata>
+                <f:viewParam name="id" value="#{DataversePage.dataverse.id}"/>
+                <f:viewParam name="alias" value="#{DataversePage.dataverse.alias}"/>
+                <f:viewParam name="ownerId" value="#{DataversePage.ownerId}"/>
+                <f:viewParam name="editMode" value="#{DataversePage.editMode}"/>
+                <f:viewAction action="#{dataverseSession.updateLocaleInViewRoot}"/>
+                <f:viewAction action="#{DataversePage.init}"/>
+                <f:viewAction action="#{dataverseHeaderFragment.initBreadcrumbs(DataversePage.dataverse)}"/>
+                <f:viewParam name="alias" value="#{SearchIncludeFragment.dataverseAlias}"/>
+                <f:viewParam name="q" value="#{SearchIncludeFragment.query}"/>
+                <f:viewParam name="types" value="#{SearchIncludeFragment.selectedTypesString}"/>
+                <f:viewParam name="fq1" value="#{SearchIncludeFragment.fq1}"/>
+                <f:viewParam name="fq0" value="#{SearchIncludeFragment.fq0}"/>
+                <f:viewParam name="fq2" value="#{SearchIncludeFragment.fq2}"/>
+                <f:viewParam name="fq3" value="#{SearchIncludeFragment.fq3}"/>
+                <f:viewParam name="fq4" value="#{SearchIncludeFragment.fq4}"/>
+                <f:viewParam name="fq5" value="#{SearchIncludeFragment.fq5}"/>
+                <f:viewParam name="fq6" value="#{SearchIncludeFragment.fq6}"/>
+                <f:viewParam name="fq7" value="#{SearchIncludeFragment.fq7}"/>
+                <f:viewParam name="fq8" value="#{SearchIncludeFragment.fq8}"/>
+                <f:viewParam name="fq9" value="#{SearchIncludeFragment.fq9}"/>
+                <f:viewParam name="sort" value="#{SearchIncludeFragment.sortField}"/>
+                <f:viewParam name="order" value="#{SearchIncludeFragment.sortOrder}"/>
+                <f:viewParam name="page" value="#{SearchIncludeFragment.page}"/>
+                <f:viewParam name="debug" value="#{SearchIncludeFragment.debug}"/>
+                <f:viewAction action="#{SearchIncludeFragment.search()}"/>
+            </f:metadata>
+
+            <h:form id="dataverseForm">
+                <!-- Edit Mode -->
+                <ui:fragment rendered="#{DataversePage.editMode == 'CREATE'}">
```

Comment:
editDataverse.xhtml - should handle both create new and edit. dataverse.xhtml should be responsible for view only